modules/modules.nix

```diff
@@ -37,6 +37,7 @@ let
     (loadModule ./misc/xdg-mime-apps.nix { condition = hostPlatform.isLinux; })
     (loadModule ./misc/xdg-user-dirs.nix { condition = hostPlatform.isLinux; })
     (loadModule ./misc/xdg.nix { })
+    (loadModule ./programs/abook.nix { })
```

Comment:
Will need `condition = hostPlatform.isLinux;` here since abook doesn't build under Darwin (see the [CI job](https://travis-ci.org/rycee/home-manager/jobs/656810727#L1264)).